.github/workflows/unit-test.yml

```
@@ -0,0 +1,12 @@
+on: [push, pull_request]
+
+jobs:
+  ut:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup go
+        uses: actions/setup-go@v4
+        with:
+          go-version: '>=1.17.0'
+      - run: make test
```

```suggestion
      - run: make test

```
Just so that git doesn't complain